src/controllers/breaches.js

```diff
@@ -56,19 +56,24 @@ async function putBreachResolution (req, res) {
   const sessionUser = req.user
   const { affectedEmail, breachId, resolutionsChecked } = req.body
   const breachIdNumber = Number(breachId)
-  const affectedEmailIsSubscriberRecord = sessionUser.primary_email === affectedEmail
-  const affectedEmailInEmailAddresses = sessionUser.email_addresses.filter(ea => ea.email === affectedEmail)
+  const affectedEmailAsSubscriber = sessionUser.primary_email === affectedEmail ? sessionUser.primary_email : false
```

Comment:
Was the const name supposed to be `affectedEmailIsSubscriber` (is instead of as?)

Reply:
So I changed the purpose of this var from a `boolean` type to a `(string | boolean)`. If email exists, then it becomes the email. This way we save a step later when referencing the email